pkg/storaged/fsys-tab.jsx

```diff
@@ -314,6 +316,16 @@ export function mounting_dialog(client, block, mode, forced_options) {
                 .then(utils.reload_systemd));
     }
 
+    let at_boot;
+    if (opt_never_auto)
+        at_boot = "never";
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17888-20221124-125406-5d04476d-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)